src/browser/constructLayoutEngine.js

```diff
@@ -205,23 +207,7 @@ export function constructLayoutEngine({
   }
 
   function handleRoutingEvent({ detail: { newUrl } }) {
-    const appsToUnmount = [];
-    const appsThatShouldBeActive = checkActivityFunctions(
-      newUrl ? strToLocation(newUrl) : location
-    );
-
-    getAppNames().forEach((app) => {
-      if (appsThatShouldBeActive.indexOf(app) < 0) {
-        appsToUnmount.push(app);
-      }
-    });
-
-    appsToUnmount.forEach((name) => {
-      if (errorParcelByAppName[name]) {
-        errorParcelByAppName[name].unmount();
```

Comment:
Error parcel unmounting is now only occurring in one place - single-spa:before-routing-event - rather than in two places. So it was removed here.